README.md

```diff
@@ -1,34 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# WebPack Template
+# JAVASCRIPT CAPSTONE
 
+Main goal of this project was working as a team with gitflow and best practices.
 
+Additionally using two different API's (TvMaze Show API & Microverse Involvement API).
+
+![screenshot](./screenshot.png)
 
 ## Built With
 
+- API
 - WEBPACK
 - HTML
-- CSS
+- SASS
 - JS
 - Bootstrap 5
 
+### Live version
+
+- live: [Javascript-Capstone](https://iwillteixeira.github.io/javascript-capstone/)
+
+### Project Video
+- [Video](https://drive.google.com/file/d/1kI4FmRN8IYxeZeRoA2kIq3AKwvdvPBeB/view)
 
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
-
 ### Prerequisites
 
 To contribute with this project you must instal the follow items:
 
--NODE
--NPM
--SASS
+- NODE
+- NPM
+- SASS
 
 ### Setup
 
 Dowload NPM and NODE
+
 ### Install
```

Comment:
[OPTIONAL] Please consider adding more detailed instructions of how to install and run the project: running dev scripts, tests, build, and deploy.